Lib/inspect.py

```diff
@@ -2736,8 +2746,9 @@ def __init__(self, parameters=None, *, return_annotation=_empty,
                     name = param.name
 
                     if kind < top_kind:
-                        msg = 'wrong parameter order: {!r} before {!r}'
-                        msg = msg.format(top_kind, kind)
+                        msg = 'wrong parameter order: {!s} parameter before {!s} parameter'
```

Comment:
Long line. I suggest to add a ruler in your editor so that you can easily see when a line is longer than 79 chars.